config_tab.py

```diff
@@ -2,9 +2,10 @@
 import json
 import mikochiku_alarm
 from PyQt5.QtWidgets import QWidget, QMainWindow
-from PyQt5.QtWidgets import QComboBox, QLabel, QFrame, QLineEdit, QPushButton
+from PyQt5.QtWidgets import QComboBox, QLabel, QFrame, QLineEdit, QPushButton, QCheckBox
 from PyQt5.QtGui     import QFont
-
+from PyQt5.QtCore import Qt
+import settings
```

Comment:
`settings` seems to be not used in this file

Reply:
I used `setting` [here](https://github.com/pusaitou/mikochiku_alarm/pull/65/files/ec05b8d958b5c9f410a180af6df2a74175395c4c#diff-5644a11635bc6732fd27268b0ae762b7R43) to set the icon.